apps/frontend/src/pages/project/libs/components/contributor-menu/contributor-menu.tsx

```diff
@@ -26,12 +34,18 @@ const ContributorMenu = ({
 	const handleMerge = useCallback(() => {
 		onMerge(contributorId);
 		onClose();
-	}, [onMerge, contributorId, onClose]);
+	}, [contributorId, onMerge, onClose]);
+
+	const handleSplit = useCallback(() => {
+		onSplit(contributorId);
+		onClose();
+	}, [contributorId, onSplit, onClose]);
 
-	const isMenuShown = hasEditPermission || hasMergePermission;
+	const isMenuShown =
+		hasEditPermission || hasMergePermission || hasSplitPermission;
 
 	return (
-		<>
+		<div className={styles["menu-container"]}>
```

Comment:
You put it in wrong spot. This div should be inside `{isMenuShown ...}` block, as we don't want to render anything in case menu shouldn't be shown